.github/workflows/release.yaml

```diff
@@ -66,6 +66,7 @@ jobs:
           gh release delete ${{ steps.version.outputs.RELEASE_VERSION }} || true
       - name: Ensure Changelog
         run: |
+          git pull --rebase
```

Comment:
```
git config user.name "OpsLevel Bots"
  git config user.email "bots@opslevel.com"
  if test -f ./.changes/v2024.5.2[3](https://github.com/OpsLevel/cli/actions/runs/9213494671/job/25348128376#step:14:3).md
  then
    echo "Skip Changie..."
  else
    go install github.com/miniscruff/changie@latest
    changie batch v202[4](https://github.com/OpsLevel/cli/actions/runs/9213494671/job/25348128376#step:14:4).5.23
    changie merge
    git add .
    git commit -m "Cut Release 'v2024.[5](https://github.com/OpsLevel/cli/actions/runs/9213494671/job/25348128376#step:14:5).23'"
    git push origin HEAD
  fi
  git tag -f v2024.5.23 -m "Cut Release 'v2024.5.23'"
  git push -f origin refs/tags/v2024.5.23
  shell: /usr/bin/bash -e {0}
```

```
[main 32b1b8b] Cut Release 'v2024.5.23'
 3 files changed, 12 insertions(+), 4 deletions(-)
 delete mode 100644 .changes/unreleased/Bugfix-20240523-114929.yaml
 create mode 100644 .changes/v2024.5.23.md
To https://github.com/OpsLevel/cli
 ! [rejected]        HEAD -> main (non-fast-forward)
error: failed to push some refs to 'https://github.com/OpsLevel/cli'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. If you want to integrate the remote changes,
hint: use 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

```

I think this might work because this diff shouldn't be detected, that changie is already processed. [source from current v2024.5.23](https://github.com/OpsLevel/cli/blob/5497ac2c20a6bc5b51db481beee93f970c5f78b4/.changes/v2024.5.23.md)